introduction.md

```diff
@@ -0,0 +1 @@
+I put some lines in the code example and merge it all up.

```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.